.travis.yml

```diff
@@ -15,14 +18,15 @@ cache:
     - ${HOME}/.npm
 
 before_install:
-  - npm install --global npm@latest
```

Comment:
I'm wondering if it's necessary to do this on Mac or Linux? If not, we can remove the whole thing.

Reply:
I think that was included to use `npm ci`, so if we test only node version that have npm >=5.7.0 it should be safe to remove.

Reply:
Let's remove it then :)